opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -94,11 +94,11 @@ message InstrumentationLibraryMetrics {
 // All DataPoint types have three common fields:
 // - Labels zero or more key-value pairs associated with the data point.
 // - StartTimeUnixNano MUST be set to the start of the interval when the
-//   descriptor Temporality includes CUMULATIVE or DELTA. This field is not set
-//   for INSTANTANEOUS timeseries, where instead the TimeUnixNano field is
-//   set for individual points.
+//   descriptor's type inlcudes a ReportStrategy. This field is not set
```

Comment:
```suggestion
//   descriptor's type includes a ReportStrategy. This field is not set
```

Reply:
Not sure about having this comment at all though. For this field in your other PR you say "For some metric types this is ignored, see MetricsDescriptor types for more".

Can you have a non-Gauge metric type that doesn't include a Report Strategy? If not, then I think the other comment is better

Reply:
We may have: RawMeasurements, not sure if others.